swatch-core/src/main/java/org/candlepin/subscriptions/db/HostRepository.java

```diff
@@ -71,11 +71,23 @@ public interface HostRepository
       left join fetch h.monthlyTotals
       where h.orgId=:orgId
         and h.instanceType='HBI_HOST'
-      order by coalesce(h.hypervisorUuid, h.subscriptionManagerId), h.hypervisorUuid, h.inventoryId, h.id
+      order by coalesce(h.hypervisorUuid, h.subscriptionManagerId), h.hypervisorUuid, h.inventoryId, h.instanceId, h.id
```

Comment:
Two thoughts here...
1. We should adjust instanceType to include "RHEL System", since we are looking to deduplicate other RHEL telemetry (this will ensure that systems brought in via metering are included). This will avoid a need to query for every new HBI system.
2. We should sort by instanceId first (see related comment around inventoryHost).

Reply:
About 1., this sounds to me like another unrelated task. 
About 2., let's discuss this in the previous comment.